client/app/telecom/telephony/alias/special/rsva/telecom-telephony-alias-special-rsva.controller.js

```diff
@@ -0,0 +1,206 @@
+
+                        return OvhApiOrder.Lexi().schema().$promise.then(function (schema) {
+                            if (schema && schema.models["telephony.NumberSpecialTypologyEnum"] && schema.models["telephony.NumberSpecialTypologyEnum"].enum) {
+                                self.typologies = _.map(_.filter(schema.models["telephony.NumberSpecialTypologyEnum"].enum, function (elt) {
+                                    return elt.match(new RegExp("^" + result.country + "_"));
+                                }), function (typo) {
+                                    return {
+                                        value: typo,
+                                        label: $translate.instant("telephony_alias_special_rsva_infos_typology_" + typo.replace(new RegExp("^" + result.country + "_"), "") + "_label")
+                                    };
+                                });
+
+                                self.rsva.typology = _.find(self.typologies, { value: result.country + "_" + specialAliasDetails.typology.replace(result.country, "") });
+                                self.rsvaForm.typology = self.rsva.typology;
+                            }
+                        });
+                    });
+                });
+            });
+        }).catch(function (err) {
+            Toast.error([$translate.instant("telephony_alias_special_rsva_loading_error"), err.message].join(" "));
+            return $q.reject(err);
+        }).finally(function () {
+            self.isLoading = false;
+        });
+    };
+
+    self.additionalInfos = function (rateCode) {
+        if (rateCode.pricePerCallWithoutTax.value) {
+            return " (" + rateCode.pricePerCallWithoutTax.text + $translate.instant("telephony_alias_special_rsva_tariff_bearing_per_call") + ")";
+        } else if (rateCode.pricePerMinuteWithoutTax.value) {
+            return " (" + rateCode.pricePerMinuteWithoutTax.text + $translate.instant("telephony_alias_special_rsva_tariff_bearing_per_minute") + ")";
+        }
+        return "";
+    };
+
+    self.onChangeTariffBearing = function () {
+        self.rateCodeChanged = !angular.equals(self.rsva.rateCode, self.rsvaForm.rateCode);
+        return self.rateCodeChanged;
+    };
+
+    self.hasChanges = function () {
+        return (self.rateCodeChanged && self.tariffBearingChangeAgreed) || !angular.equals(self.rsva.typology, self.rsvaForm.typology);
+    };
+
+    self.applyChanges = function () {
+        self.isUpdating = true;
+
+        return OvhApiTelephony.Rsva().Lexi().scheduleRateCode({
+            billingAccount: $stateParams.billingAccount,
+            serviceName: $stateParams.serviceName
+        }, { rateCode: self.rsvaForm.rateCode.code || "" }).$promise.then(function () {
+            self.rsva = angular.copy(self.rsvaForm);
+            self.isEditing = false;
+
+            return OvhApiTelephony.Rsva().Lexi().edit({
```

Comment:
⚠️ Couldn't be tested I guess.

You have to bump `ovh-api-services` to `v3.0.0` (in progress here: #660) and update this call to:

```js
return OvhApiTelephony.Rsva().v6().edit({
    …
})
```